app/server.js

```diff
@@ -13,7 +13,7 @@ const server = new Hapi.Server({
   }
 })
 
-server.connection({port: process.env.PORT || 5000, routes: {cors: true}})
+server.connection({port: process.env.PORT || 27606, routes: {cors: true}})
```

Comment:
Updated the Hapi server as well, so log output such as
```
app_1  | Server Running at: http://6fca9108133e:27606
```
reads consistently, so as to not confuse anyone.